test/flanders/markdown_test.clj

```diff
@@ -0,0 +1,22 @@
+(ns flanders.markdown-test
+  (:require [clojure.test :refer [deftest is]]
+            [flanders.core :as f]
+            [flanders.markdown :as f.markdown]))
+
+(deftest signuature-type->markdown
+  (is (= (f.markdown/->markdown (f/sig :parameters []))
+         "() => Anything\n"))
+
+  (is (= (f.markdown/->markdown (f/sig :parameters [(f/int)]))
+         "(Integer) => Anything\n\n"))
+
+  (is (= (f.markdown/->markdown (f/sig :parameters [(f/int) (f/str)]))
+         "(Integer,  String) => Anything\n\n\n"))
+
+  (is (= (f.markdown/->markdown (f/sig :parameters [(f/int)]
+                                       :rest-parameter (f/str)))
+         "(Integer,  String ...) => Anything\n\n\n"))
+
+  (is (= (f.markdown/->markdown (f/sig :parameters [(f/int)]
+                                       :rest-parameter (f/str)))
+         "(Integer,  String ...) => Anything\n\n\n")))
```

Comment:
Why the number of carriage returns are not the same?

Reply:
I was wondering the same thing! 😅  

I haven't investigated it but my suspicion is that it has something to do with the total number of renderable nodes. One for the first parameter list, one for the rest parameter, and one for the return. Above you'll notice there are two returns for the case where there is one parameter, and one return. But, again, it's a guess. I'll look at it more closely and see if I can give you a better answer. 😄